packages/react-ui-map/src/Mapbox/Mapbox.js

```diff
@@ -28,36 +26,39 @@ export default class Mapbox extends PureComponent {
 
   constructor(props, context) {
     super(props, context)
+    mapboxgl.accessToken = props.token
 
     this.state = {
       map: null,
     }
   }
 
   getChildContext() {
-    return { map: this.state.map }
+    return {
+      map: this.state.map,
+    }
   }
 
   componentDidMount() {
-    mapboxgl.accessToken = this.props.token
-    const map = new mapboxgl.Map({
-      container: this.container,
-      style: this.props.style,
-      center: this.props.center,
-      zoom: this.props.zoom,
-      theme: this.props.theme,
-    })
-
-    map.on('load', () => {
-      this.setState({ map })
-    })
+    if (this.props.center) {
```

Comment:
So if there is no `center` prop we don't render the map?

Reply:
so i'm confused, why render the map if we don't have a location?

Reply:
also it breaks if no center is provided.

Reply:
Why can't we just have a default prop for `center`? If you don't pass it in, you get the default e.g. Atlanta.

Reply:
From the [docs](https://www.mapbox.com/mapbox-gl-js/api/#map)
> The inital geographical centerpoint of the map. If  center is not specified in the constructor options, Mapbox GL JS will look for it in the map's style object. If it is not specified in the style, either, it will default to  [0, 0] Note: Mapbox GL uses longitude, latitude coordinate order (as opposed to latitude, longitude) to match GeoJSON.